Bottom: 017592d95040e1620bd6e0ce23ec237e2cef38e8
Top:    b34661bd897e6a163b13168475e826b7edaf7ede
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-26 13:02:04 -0500

Refresh of all-docker_volume

---
diff --git a/roles/docker-mantisbt/tasks/main.yml b/roles/docker-mantisbt/tasks/main.yml
index 3547b88c..a8ff7fa4 100644
--- a/roles/docker-mantisbt/tasks/main.yml
+++ b/roles/docker-mantisbt/tasks/main.yml
@@ -158,6 +158,7 @@
     detach: true
     restart_policy: '{{ docker_restart_policy }}'
     hostname: '{{ container_addr_map.mantisbt.hostname }}'
+    hostname: '{{ mantisbt_hostname }}'
     ports: '{{ container_port_map.mantisbt.port_args }}'
     link: 
       -'{{ openldap_container_name }}:mantisbt_ldap'
@@ -167,10 +168,10 @@
       - '{{ openssl_dv_name }}'
       - '{{ mantisbt_dv_name }}'
     env: 
-      - MANTISBT_HOSTNAME={{ container_addr_map.mantisbt.hostname }}
-      - MANTISBT_MAIL_USER=mantis
-      - MANTISBT_MAIL_PASSWORD={{ mantisbt_email_password | quote }}
-      - MANTISBT_LDAP_PASSWORD={{ ldap_proxyagent_password | quote }}
+      MANTISBT_HOSTNAME: '{{ container_addr_map.mantisbt.hostname }}'
+      MANTISBT_MAIL_USER: 'mantis'
+      MANTISBT_MAIL_PASSWORD: '{{ mantisbt_email_password | quote }}'
+      MANTISBT_LDAP_PASSWORD: '{{ openldap_proxyagent_password | quote }}'
     image: '{{ mantisbt_image_name }}:{{ docker_image_tag }}'
 
 # *****************************************************************************
